content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -30,24 +30,24 @@ influx write [command]
-| `-c` | `--active-config`  | CLI configuration to use for command                                  | string     |                      |
-| `-b` | `--bucket`         | Bucket name                                                           | string     | `INFLUX_BUCKET_NAME` |
-|      | `--bucket-id`      | Bucket ID                                                             | string     | `INFLUX_BUCKET_ID`   |
-|      | `--configs-path`   | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) | string     |`INFLUX_CONFIGS_PATH` |
-|      | `--debug`          | Output errors to stderr                                               |            |                      |
-|      | `--encoding`       | Character encoding of input (default `UTF-8`)                         | string     |                      |
-| `-f` | `--file`           | File to import                                                        | string     |                      |
-|      | `--format`         | Input format (`lp` or `csv`, default `lp`)                            | string     |                      |
-|      | `--header`         | Prepend header line to CSV input data                                 | string     |                      |
-| `-h` | `--help`           | Help for the `dryrun` command                                         |            |                      |
-|      | `--host`           | HTTP address of InfluxDB (default `http://localhost:8086`)            | string     | `INFLUX_HOST`        |
-| `-o` | `--org`            | Organization name                                                     | string     | `INFLUX_ORG`         |
-|      | `--org-id`         | Organization ID                                                       | string     | `INFLUX_ORG_ID`      |
-| `-p` | `--precision`      | Precision of the timestamps (default `ns`)                            | string     | `INFLUX_PRECISION`   |
-|      | `--skipHeader`     | Skip first n rows of input data                                       | integer    |                      |
-|      | `--skipRowOnError` | Output CSV errors to stderr, but continue processing                  |            |                      |
-|      | `--skip-verify`    | Skip TLS certificate verification                                     |            |                      |
-| `-t` | `--token`          | Authentication token                                                  | string     | `INFLUX_TOKEN`       |
-| `-u` | `--url`            | URL to import data from                                               | string     |                      |
+| Flag |                    | Description                                                           | Input type | {{< cli/mapped >}}    |
+|:-----|:-------------------|:----------------------------------------------------------------------|:----------:|:----------------------|
```

Comment:
looks like `--active-config` disappeared

Reply:
I don't see --active-config listed in the currently live docs: https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/write/.

Reply:
Gotcha, the --active-config flag is part of RC added to beta-17 branch. We'll want to retain when resolving this merge conflict/ pushing changes to beta-17.